src/utils/internal/parseMultipartData.ts

```diff
@@ -53,7 +53,7 @@ export function parseMultipartData<T extends DefaultRequestMultipartBody>(
     return undefined
   }
 
-  const [, ...directives] = contentType.split('; ')
+  const [, ...directives] = contentType.split(/; */)
```

Comment:
This is a great change! Let's add a test for handling such headers in `src/utils/internal/parseMultipartData.test.ts`. Please, would you be comfortable with adding such test?